img/u/uclahealth.org.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"	 viewBox="0 0 180 180" enable-background="new 0 0 180 180"><path fill="#2874AE" opacity="1.000000" stroke="none" 	d="M119.000000,181.000000 	C79.333336,181.000000 40.166672,181.000000 1.000005,181.000000 	C1.000004,121.000008 1.000004,61.000011 1.000002,1.000013 	C60.999989,1.000009 120.999977,1.000009 180.999969,1.000004 	C180.999985,60.999985 180.999985,120.999969 181.000000,180.999969 	C160.500000,181.000000 140.000000,181.000000 119.000000,181.000000 M146.437088,43.008465 	C146.866837,38.092110 147.296570,33.175751 147.778931,27.657228 	C140.695480,27.657228 134.062759,27.888863 127.456398,27.571583 	C123.401588,27.376844 122.153969,28.657961 121.814919,32.768085 	C119.707512,58.315453 117.487381,83.858047 114.843529,109.354416 	C113.781403,119.597237 109.140282,128.149048 99.218803,132.735901 	C81.256660,141.040070 65.156067,130.999573 64.883369,111.378700 	C64.853310,109.215309 64.869545,107.039436 65.060204,104.886696 	C66.528877,88.303703 68.062866,71.726501 69.541161,55.144344 	C70.345818,46.118397 71.081352,37.086285 71.881981,27.657455 	C64.567902,27.657455 57.914291,27.879362 51.285046,27.573380 	C47.372410,27.392784 46.198807,28.739588 45.929382,32.577274 	C44.938515,46.691364 43.406582,60.767193 42.391979,74.879967 	C41.366081,89.149895 39.838886,103.462616 40.100994,117.727745 	C40.490662,138.935120 55.156925,155.368607 75.331474,159.121429 	C107.217667,165.052811 135.601654,144.794540 139.855133,112.021690 	C142.788162,89.422798 144.206650,66.627319 146.437088,43.008465 z"/><path fill="#FDFEFE" opacity="1.000000" stroke="none" 	d="M146.369537,43.464081 	C144.206650,66.627319 142.788162,89.422798 139.855133,112.021690 	C135.601654,144.794540 107.217667,165.052811 75.331474,159.121429 	C55.156925,155.368607 40.490662,138.935120 40.100994,117.727745 	C39.838886,103.462616 41.366081,89.149895 42.391979,74.879967 	C43.406582,60.767193 44.938515,46.691364 45.929382,32.577274 	C46.198807,28.739588 47.372410,27.392784 51.285046,27.573380 	C57.914291,27.879362 64.567902,27.657455 71.881981,27.657455 	C71.081352,37.086285 70.345818,46.118397 69.541161,55.144344 	C68.062866,71.726501 66.528877,88.303703 65.060204,104.886696 	C64.869545,107.039436 64.853310,109.215309 64.883369,111.378700 	C65.156067,130.999573 81.256660,141.040070 99.218803,132.735901 	C109.140282,128.149048 113.781403,119.597237 114.843529,109.354416 	C117.487381,83.858047 119.707512,58.315453 121.814919,32.768085 	C122.153969,28.657961 123.401588,27.376844 127.456398,27.571583 	C134.062759,27.888863 140.695480,27.657228 147.778931,27.657228 	C147.296570,33.175751 146.866837,38.092110 146.369537,43.464081 z"/></svg>
```

Comment:
Can you try to find a higher-resolution logo? I tried, but [the Box folder](https://www.uclahealth.org/brand/logos/ucla-health-logo) that UCLA Health uses wasn't allowing me to access anything.

Reply:
@hkamran80 I tried too, but it insists I create a Box account.

What I used was https://www.uclahealth.org/themes/custom/uclahealth/favicons/safari-pinned-tab.svg which is in [uclahealth.org](https://www.uclahealth.org/)'s headers.

Reply:
In that case, it's fine.

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 180 180"><path fill="#2874AE" d="M119 181H1V1h180v180h-62m27-138 2-15h-21c-4-1-5 1-5 5l-7 76c-1 11-6 19-16 24-18 8-34-2-34-22v-6l5-50 2-27H51c-4-1-5 1-5 5l-4 42-2 43c0 21 15 37 35 41 32 6 61-14 65-47l6-69z"/><path fill="#FDFEFE" d="m146 43-6 69c-4 33-33 53-65 47-20-4-35-20-35-41l2-43 4-42c0-4 1-6 5-5h21l-2 27-5 50v6c0 20 16 30 34 22 10-5 15-13 16-24l7-76c0-4 1-6 5-5h21l-2 15z"/></svg>
```